### PR TITLE
fix(ingestion/looker):  skip personal folder independent looks

### PR DESCRIPTION
**Pull request by sid-acryl**

### CODE REVIEW

File: metadata-ingestion/tests/integration/looker/test_looker.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import json
 import time
 from datetime import datetime
+from pathlib import Path
 from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, cast
 from unittest import mock
 
 import pytest
+from _pytest.config import Config
```
Comment (hsheth2) [inline metadata-ingestion/tests/integration/looker/test_looker.py:9]: why `_pytest`?

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py [hunk 2]
```diff
@@ -286,7 +286,8 @@ def _get_views_from_fields(self, fields: List[str]) -> List[str]:
             except AssertionError:
                 self.reporter.report_warning(
                     title="Failed to Extract View Name from Field",
-                    message="The field was not prefixed by a view name. This can happen when the field references another dynamic field.",
+                    message="The field was not prefixed by a view name. This can happen when the field references "
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py:289]: this reformatting stuff that your IDE is doing makes it extremely hard to review PRs. Please fix your setup

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py [hunk 3]
```diff
@@ -1486,13 +1488,27 @@ def extract_independent_looks(self) -> Iterable[MetadataWorkUnit]:
...
 
+            if self.source_config.skip_personal_folders:
+                if look.folder is not None and (
+                    look.folder.is_personal or look.folder.is_personal_descendant
+                ):
+                    self.reporter.info(
+                        title="Dropped Look",
+                        message="Dropped due to being a personal folder",
+                        context=f"Look ID: {look.id}",
+                    )
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py:1506]: does this reporting match what we do for other dashboards/looks in personal folders?

**Review by hsheth2 — CHANGES_REQUESTED** [review]
Mostly looks good, just a few minor comments

File: metadata-ingestion/tests/integration/looker/test_looker.py (hunk 1 above)
Comment (sid-acryl) [reply metadata-ingestion/tests/integration/looker/test_looker.py:9]: It is to silent typing lint issue for function `ingest_independent_looks`

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py (hunk 2 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py:289]: it was pep8 max line warning in pycharm. I disabled it

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py (hunk 3 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py:1506]: yes

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py [hunk 4]
```diff
@@ -1486,13 +1487,27 @@ def extract_independent_looks(self) -> Iterable[MetadataWorkUnit]:
...
 
             if look.query_id is None:
                 logger.info(f"query_id is None for look {look.title}({look.id})")
                 continue
 
+            if self.source_config.skip_personal_folders:
+                if look.folder is not None and (
+                    look.folder.is_personal or look.folder.is_personal_descendant
+                ):
+                    self.reporter.info(
```
Comment (treff7es) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py:1501]: What is the benefit of having this reporter info vs collecting these in the report below and add a debug log?

**Review by treff7es — APPROVED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py (hunk 4 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py:1501]: This is to maintain consistency with the logic used in the personal dashboards.

**Pull request merged**